src/main.c

```diff
@@ -58,7 +58,10 @@ static void draw_row_callback(GContext *ctx, Layer *cell_layer, MenuIndex *cell_
 }
 
 static int16_t get_cell_height_callback(struct MenuLayer *menu_layer, MenuIndex *cell_index, void *context) {
-  return CHECKBOX_WINDOW_CELL_HEIGHT;
+  return PBL_IF_ROUND_ELSE(
+    menu_layer_is_index_selected(menu_layer, cell_index) ? 
+      MENU_CELL_ROUND_FOCUSED_SHORT_CELL_HEIGHT : MENU_CELL_ROUND_UNFOCUSED_TALL_CELL_HEIGHT,
+    44);
```

Comment:
Do we need to worry about this magic 44? Should it just be CHECKBOX_WINDOW_CELL_HEIGHT as before?